utils/metrics/src/lib.rs

```diff
@@ -26,15 +36,22 @@ pub async fn init_prometheus(
 }
 
 async fn collect_metrics() -> impl IntoResponse {
-    let registry = prometheus::default_registry();
-    let metric_families = registry.gather();
+    let default_registry = prometheus::default_registry();
+    let metric_families = default_registry.gather();
     let mut metrics = vec![];
 
     let encoder = TextEncoder::new();
     encoder
         .encode(&metric_families, &mut metrics)
         .expect("Encoding Prometheus metrics must succeed.");
 
+    for (name, registry) in REGISTRIES_EXT.read().await.iter() {
+        metrics.extend_from_slice(format!("\n# SECTION {name}\n").as_bytes());
```

Comment:
Is this `SECTION` a standard Prometheus keyword? Does Grafana handle it? If not, I'd not add it.

Reply:
I believe the leading `#` makes it a comment line, gGafana should just ignore it
https://prometheus.io/docs/prometheus/latest/querying/basics/#comments

Reply:
Thanks. Still, I wouldn't add custom stuff there. Grafana may be fine, I encountered parsers that were not happy with such things even though it was technically legal to add them. It also doesn't ignore it, e.g., those `# HELP` and `# TYPE` are used.

Reply:
OK, let me remove it, blank lines should be fine?

Reply:
I would not add anything, vanilla metrics are the safest option.

Reply:
Thank you!